components/scream/src/physics/p3/tests/p3_run_and_cmp.cpp

```diff
@@ -207,14 +241,20 @@ int main (int argc, char** argv) {
     std::cout <<
       argv[0] << " [options] baseline-filename\n"
       "Options:\n"
-      "  -g        Generate baseline file.\n"
-      "  -f        Use fortran impls instead of c++.\n"
-      "  -t <tol>  Tolerance for relative error.\n";
+      "  -g          Generate baseline file. Default False.\n"
+      "  -f          Use fortran impls instead of c++. Default False.\n"
+      "  -t <tol>    Tolerance for relative error. Default 0.\n";
+      "  -s <steps>  Number of timesteps. Default=6.\n";
+      "  -i <cols>   Number of columns. Default=6.\n";
+      "  -r <repeat> Number of repititions, implies timing run (generate + no I/O). Default=0.\n";
```

Comment:
Typo: rep**e**titions

Reply:
Looks like the automerger got to it first. I will fix this typo in a subsequent pr.